src/nuget/xunit.netcore.extensions.nuspec

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>xunit.netcore.extensions</id>
+    <version>1.0.0-prerelease</version>
+    <title>xUnit extensions of .NET Core test projects</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=518631</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>xUnit extensions of .NET Core test projects</summary>
+    <description>This package provides things like various traits and discovers like OuterLoop/ActiveIssue that are used by .NET Core framework test projects.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <dependencies>
+      <dependency id="System.Diagnostics.Contracts" version="4.0.0-beta-22703" />
```

Comment:
I assume this one isn't actually necessary now after the recent change from Contract.Assert to Debug.Assert.